packages/core/src/tools/browserDetection.ts

```diff
@@ -3,5 +3,5 @@ export function isIE() {
 }
 
 export function isChromium() {
-  return !!(window as any).chrome
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏if `window.chrome === false` in headless, what about `'chrome' in window`?

Reply:
my bad it is actually `undefined`